### DISCUSSION TITLE
Correction on "NetBeans Platform Quick Start Using Maven"

### DISCUSSION THREAD

**Discussion by ProfTuan** [apache/netbeans#3892] [Ideas]
Hi,

I wasn't sure where to report edits for content on the website. I want to report on the following page:

https://netbeans.apache.org/tutorials/nbm-maven-quickstart.html

The section tilted "Creating Your First Maven Based NetBeans Module" should specify where to click to start the process for creating a new module. Ideally it should indicate that the user should right click the "Modules" folder of the _parent_ module, then select "Create new module". The way the tutorial is written is it would create an independent netbeans module without the annotations to the pom files. I hope this make sense and I hope this is the proper platform to report on the issues with the tutorials.

**Comment by mbien** [comment]
hi,
simply scroll down to the end of the page you want to edit and follow the "See this page in GitHub." link. After that you can edit it with the pencil button. Once you are done, create a pull request (select "Create a new branch for this commit and start a pull request") so that someone can review/integrate it properly.
